src/ResourceManager/Profile/Commands.Profile.Test/EnvironmentCmdletTests.cs

```diff
@@ -105,6 +105,7 @@ public void AddsAzureEnvironmentUsingARMEndpoint()
             Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.ResourceManager], cmdlet.ResourceManagerEndpoint);
             Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.ActiveDirectory], cmdlet.ActiveDirectoryEndpoint);
             Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.ActiveDirectoryServiceEndpointResourceId], cmdlet.ActiveDirectoryServiceEndpointResourceId);
+            Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.GraphEndpointResourceId], cmdlet.GraphAudience);
```

Comment:
I don't think I understand these assertions.  How are these verifying that the value from the metadata endpoint is being used?